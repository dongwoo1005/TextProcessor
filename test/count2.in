stdin count 5 count 5
one5 two5 three5 four5 five5 six5 seven5 eight5 nine5 ten5
