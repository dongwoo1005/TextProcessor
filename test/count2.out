one1
two2
three3
four4
five1
six6
seven7
eight8
nine9
ten10
